api/src/routers/meals.js

```diff
@@ -0,0 +1,80 @@
+import express from "express";
+import db from "../database_client.js";
+
+// This router can be deleted once you add your own router
+const mealsRouter = express.Router();
+
+mealsRouter.get("/", (req, res) => {
+  res.json({ message: "Hello meals router" });
+});
+
+
+//future-meals Router
+mealsRouter.get("/future-meals", async (req, res) => {
+  try{
+    const result = await db.raw("SELECT * FROM meal WHERE `when` > NOW();");
+    const futureMeals = result[0]; 
```

Comment:
Very good job on adding this `futureMeals` variable! It makes the code so much easier to understand! This applies to all endpoints, as I see that you used such a variable in all methods. 🤩